SPECS/python-apache-libcloud/python-apache-libcloud.spec

```diff
@@ -0,0 +1,1064 @@
+#
+# spec file for package python-apache-libcloud
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+Name:           python-apache-libcloud
+Version:        3.5.1
+Release:        2
+Summary:        Abstraction over multiple cloud provider APIs
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://libcloud.apache.org
+Source0:        http://archive.apache.org/dist/libcloud/apache-libcloud-%{version}.tar.bz2#/apache-libcloud-%{version}.tar.gz
```

Comment:
We shouldn't rename this to a `.tar.gz` file if we're downloading a `.tar.bz2` file

Reply:
Done, using https github mirror for sources.